electrochemistry.ttl

```diff
@@ -1115,38 +1139,46 @@ charge passed at infinite time, D is the diffusion coefficient of the electroact
 
 ###  http://emmo.info/electrochemistry#electrochemistry_402598d9_0eea_405d_b0d0_3b8b79deba6f
 :electrochemistry_402598d9_0eea_405d_b0d0_3b8b79deba6f rdf:type owl:Class ;
-                                                       rdfs:subClassOf :electrochemistry_0a0de817_addc_46a5_8ba2_255d48cdf366 ;
+                                                       rdfs:subClassOf :electrochemistry_0a0de817_addc_46a5_8ba2_255d48cdf366 ,
+                                                                       [ rdf:type owl:Restriction ;
+                                                                         owl:onProperty emmo:EMMO_ae2d1a96_bfa1_409a_a7d2_03d69e8a125a ;
+                                                                         owl:someValuesFrom :electrochemistry_0f007072-a8dd-4798-b865-1bf9363be627
+                                                                       ] ;
+                                                       emmo:EMMO_b432d2d5_25f4_4165_99c5_5935a7763c1a "solid electrolyte interphase (SEI) growth"@en ;
```

Comment:
yes, this one is really tricky especially since some processes like LiPlating are DegradationPhenomenon in some cells (e.g. Li-ion) but essential to the operation of other cells (e.g. anode-less Li metal).